doc/howto/build-linux.txt

```diff
@@ -68,18 +68,18 @@ To build X-Ray Engine you'll need following libraries:
 * zlib: http://zlib.net
 All mentioned libraries are already in repository as submodules!
 
-Setup:
+Usage:
 ------
 - Install S.T.A.L.K.E.R Call of Pripyat using wine.
 For example if you have a cd-rom you can type `wine setup.exe` to start installation. 
-It's recommended do not use path with spaces, for instance you can use "c:\cop" as 
+Having spaces in path is not recommended, for instance you can use "c:\cop" as 
```

Comment:
Why is having spaces in path not recommended?

Reply:
Probably there were some troubles with them that now have been fixed, dunno.